src/upgrades/mongodb_upgrade.py

```diff
@@ -112,10 +140,45 @@ def _on_force_upgrade_action(self, event: ActionEvent) -> None:
             return
         logger.debug("Forcing upgrade")
         event.log(f"Forcefully upgrading {self.unit.name}")
```

Comment:
```suggestion
        event.log(f"Forcefully upgrading {self.charm.unit.name}")
```

In addition to this one, a few lines above, `self.app.name` should be changed to `self.charm.app.name`